src/com/artos/framework/infra/GuiSelector.java

```diff
@@ -0,0 +1,383 @@
+/*******************************************************************************
```

Comment:
You changed the file name. What is the purpose and is it necessory?

Reply:
I think the GUI test selector is quite essential, and I'm not sure if refactoring it is okay, so I just leave the original unchanged and created a new class to simulate it.
Because I'm tring to control the interface through code, so I changed the button and table to public. Other refectorings are basically remained the same funtion.
My understanding is that the GUITestSelector is a internal class not exposed to the Artos users, so we can refector it without harm to the user projects. Is this correct?